datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -5863,3 +5863,47 @@ ORDER BY k;
 ----
 1 1.8125 6.8007813 Float16 Float16
 2 8.5 8.5 Float16 Float16
+
+# The result is 0.19432323191699075 actually
+query R
+SELECT kurtosis_pop(col) FROM VALUES (1), (10), (100), (10), (1) as tab(col);
+----
+0.194323231917
```

Comment:
I tried this function with the CLI
```
DataFusion CLI v41.0.0
> SELECT kurtosis_pop(col) FROM VALUES (1), (10), (100), (10), (1) as tab(col);
+-----------------------+
| kurtosis_pop(tab.col) |
+-----------------------+
| 0.19432323191699075   |
+-----------------------+
````
I'm not sure but I guess the sqllogicttest may do some rounds for the result.

Reply:
Yes sqllogictest will do the rounding according to https://github.com/apache/datafusion/tree/main/datafusion/sqllogictest
```
floating point values are rounded to the scale of "12",
```